Vagrantfile

```diff
@@ -81,13 +87,91 @@ Vagrant.configure(2) do |config|
       "hpccloud" => ["hpccloud"],
       "pyfr" => ["hpccloud"]
     }
+
     ansible.verbose = "vv"
     ansible.extra_vars = {
       default_user: "vagrant",
-      development: dev
+      development: dev,
+      hpccloud_password: hpccloud_password
     }
-
     ansible.playbook = "ansible/site.yml"
+  end
+
+  # Should we also deploy a "cluster" on the VM
+  if demo
+    # Create temp directory
+    dir = Dir.mktmpdir()
+
+    # First download cumulus so we have access to the playbooks
+    config.vm.provision "ansible" do |ansible|
+
+      # If there is a better way to run playbook aganst localhost let
+      # me know!
+      ansible.raw_arguments = ['--limit=localhost',
+        '--inventory-file=demo/inventory/localhost',
```

Comment:
If you want you can pass in '--inventory-file="localhost,"'  (with the trailing comma) and remove the demo/inventory/localhost file.